cluster/src/main/java/com/hartwig/pipeline/tertiary/peach/Peach.java

```diff
@@ -0,0 +1,89 @@
+package com.hartwig.pipeline.tertiary.peach;
+
+import java.util.List;
+
+import com.hartwig.pipeline.Arguments;
+import com.hartwig.pipeline.ResultsDirectory;
+import com.hartwig.pipeline.execution.PipelineStatus;
+import com.hartwig.pipeline.execution.vm.BashCommand;
+import com.hartwig.pipeline.execution.vm.BashStartupScript;
+import com.hartwig.pipeline.execution.vm.InputDownload;
+import com.hartwig.pipeline.execution.vm.VirtualMachineJobDefinition;
+import com.hartwig.pipeline.execution.vm.VirtualMachinePerformanceProfile;
+import com.hartwig.pipeline.execution.vm.VmDirectories;
+import com.hartwig.pipeline.execution.vm.python.Python3Command;
+import com.hartwig.pipeline.metadata.SomaticRunMetadata;
+import com.hartwig.pipeline.report.EntireOutputComponent;
+import com.hartwig.pipeline.report.Folder;
+import com.hartwig.pipeline.report.RunLogComponent;
+import com.hartwig.pipeline.resource.ResourceFiles;
+import com.hartwig.pipeline.stages.Stage;
+import com.hartwig.pipeline.storage.GoogleStorageLocation;
+import com.hartwig.pipeline.storage.RuntimeBucket;
+import com.hartwig.pipeline.tertiary.purple.PurpleOutput;
+import com.hartwig.pipeline.tools.Versions;
+
+public class Peach implements Stage<PeachOutput, SomaticRunMetadata> {
+    static final String NAMESPACE = "peach";
+    private final InputDownload purpleGermlineVcfDownload;
+    private final ResourceFiles resourceFiles;
+
+    public Peach(final PurpleOutput purpleOutput, final ResourceFiles resourceFiles) {
+        purpleGermlineVcfDownload = new InputDownload(purpleOutput.outputLocations().germlineVcf());
+        this.resourceFiles = resourceFiles;
+    }
+
+    @Override
+    public List<BashCommand> inputs() {
+        return List.of(purpleGermlineVcfDownload);
+    }
+
+    @Override
+    public String namespace() {
+        return NAMESPACE;
+    }
+
+    @Override
+    public List<BashCommand> commands(final SomaticRunMetadata metadata) {
+        return List.of(new Python3Command("peach",
+                Versions.PEACH,
+                "src/main.py",
```

Comment:
perhaps we can ask him to create a wrapper script so we don't access a source file directly? (although im ok to ask for that in his next release)